libs/shared/barista-definitions/tsconfig.spec.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I guess we can remove this file as these type definitions will never contain tests